include/z64save.h

```diff
@@ -464,80 +464,82 @@ typedef enum LinkAge {
  * SaveContext.eventChkInf
  */
 
-#define EVENTCHKINF_02 0x02
-#define EVENTCHKINF_03 0x03
-#define EVENTCHKINF_04 0x04
-#define EVENTCHKINF_05 0x05
-#define EVENTCHKINF_07 0x07
-#define EVENTCHKINF_09 0x09
-#define EVENTCHKINF_0A 0x0A
+#define EVENTCHKINF_FIRST_SPOKE_TO_MIDO 0x02
+#define EVENTCHKINF_COMPLAINED_ABOUT_MIDO 0x03
+#define EVENTCHKINF_SHOWED_MIDO_SWORD_SHIELD 0x04
+#define EVENTCHKINF_DEKU_TREE_OPENED_MOUTH 0x05
+#define EVENTCHKINF_OBTAINED_KOKIRI_EMERALD_DEKU_TREE_DEAD 0x07
+#define EVENTCHKINF_USED_DEKU_TREE_BLUE_WARP 0x09
+#define EVENTCHKINF_PLAYED_SARIAS_SONG_FOR_MIDO_AS_ADULT 0x0A
 #define EVENTCHKINF_0B 0x0B
-#define EVENTCHKINF_0C 0x0C
-#define EVENTCHKINF_0F 0x0F
+#define EVENTCHKINF_MET_DEKU_TREE 0x0C
+#define EVENTCHKINF_SPOKE_TO_MIDO_ABOUT_SARIA 0x0F
 #define EVENTCHKINF_TALKED_TO_MALON_FIRST_TIME 0x10
-#define EVENTCHKINF_11 0x11
+#define EVENTCHKINF_SPOKE_TO_INGO_AT_RANCH_BEFORE_TALON_RETURNS 0x11
 #define EVENTCHKINF_RECEIVED_WEIRD_EGG 0x12
 #define EVENTCHKINF_TALON_WOKEN_IN_CASTLE 0x13
 #define EVENTCHKINF_TALON_RETURNED_FROM_CASTLE 0x14
 #define EVENTCHKINF_TALKED_TO_CHILD_MALON_AT_RANCH 0x15
 #define EVENTCHKINF_CAN_LEARN_EPONAS_SONG 0x16
 #define EVENTCHKINF_EPONA_OBTAINED 0x18
-#define EVENTCHKINF_1B 0x1B
-#define EVENTCHKINF_1C 0x1C
-#define EVENTCHKINF_1D 0x1D
+#define EVENTCHKINF_OBTAINED_KOKIRI_EMERALD 0x19
```

Comment:
Is this even a real flag?

Reply:
This and the other flag were pulled in from SoH. Not that they see much real use, but they're there.

I'll revert this one only because of skepticism considering there's another flag for this, and this flag is never used.

Reply:
i would say that `0x07` and `0x09` are related to that event
```
// z_door_warp1.c:507
Flags_SetEventChkInf(EVENTCHKINF_07);
Flags_SetEventChkInf(EVENTCHKINF_09);
Item_Give(play, ITEM_KOKIRI_EMERALD);
```

in one of my fork's branches with some documentation process it looks like that:
![image](https://github.com/user-attachments/assets/c79fc36b-ae91-45c1-9c79-54e03e274aae)

Reply:
If they do exist, where are they referenced? Currently no define exists for them because it is never referenced in code, unless I am missing something.
If it doesnt ever get used in code, how can we know what its name should be?

Reply:
I can confirm that it isn't a real flag, it isn't even referenced anywhere in SoH either.

Reply:
Thanks.
This goes back to my comment in my previous review, it would be helpful if each flag is carefully examined for accuracy, instead of just blindly copying from SOH.